Cargo.lock

```diff
@@ -4,9 +4,9 @@ version = 3
 
 [[package]]
 name = "addr2line"
-version = "0.17.0"
+version = "0.20.0"
```

Comment:
In general, why are there any revisions at all to Cargo.lock?  This should not change at all.

Reply:
A few dependencies I've added to stacks-core are using their latest version and that might be causing this. When / if we reorganize our dependencies to be defined in the workspace level this should happen less often.

That said, with this background do you still consider it a problem?

Reply:
Yes. Please revert them.  I audit packages that we add as dependencies.